src/Stages.js

```diff
@@ -79,10 +74,16 @@ class Stages extends Component {
     }) 
   }
 
-  filterByUniverse(universe) {
-    let filteredStages = this.state.stageList.filter((stage) => {  
-      return stage.universe.name === universe
-    })
+  filterByUniverse = (universe) => {
+    let filteredStages;
```

Comment:
Thank you for adding the all, was seeing that issue as well when filtering. Great add!